### DISCUSSION TITLE
Publish to microsoft store in msix format

### DISCUSSION THREAD

**Discussion by soredake** [doublecmd/doublecmd#1101] [Suggestions]
### Description

Example benefits of msix package format: https://github.com/felixrieseberg/windows95/issues/221 https://learn.microsoft.com/en-us/windows/msix/overview#key-features

**Comment by alexx2000** [comment]
Publishing in the Microsoft Store requires codesign certificate which we don't have.

**Comment by soredake** [comment]
@alexx2000 i'm not sure about this: https://www.advancedinstaller.com/msix-publish-microsoft-store.html
>Automatic Digital Signing - MSIX requires[ digital signing](https://www.advancedinstaller.com/msix-digital-signing.html) and, by publishing it to Microsoft Store, your app gets signed automatically. Therefore, you don’t need to buy a third-party CA (certification authority), and your app will have the credibility of being secure in case you use your own certification.

**Comment by alexx2000** [comment]
> To publish your app to the store, you need to have a developer account. Go to Microsoft Partner Center and sign up.
> 
> There is a one-time registration fee, and it depends on what type of account you are creating:
> 
> 19$ for an individual account
> 
> 99$ for a company account

**Comment by soredake** [comment]
@alexx2000 it's not so much for one time payment, i can probably donate this amount in the future.

**Comment by JonnyTech** [comment]
A big limitation of store apps is the restrictions imposed by Microsoft. Running UWP applications as an Administrator cannot be done. Ths is a limitation of the Universal Windows Platform. Running a UWP application as an Administrator will NOT grant it elevated permissions like a Win32 desktop application. For a file manager that is used by many power users, this is a major annnoyance. We will get many issues of people complaining that DC has bugs.

**Comment by alexx2000** [comment]
Microsoft Store supports publishing native Win32 applications now. But I don't know how it looks.

**Comment by soredake** [comment]
@JonnyTech first, doublecmd is not UWP app, second, elevation is supported in [UWP ](https://github.com/microsoft/WindowsAppSDK/issues/896#issuecomment-1149004269) and [msix](https://www.advancedinstaller.com/allow-elevation-msix-packages.html).

I've successfully run files (uwp app) and navigated to location that requires elevation in windows explorer (C:\Windows\System32\config\systemprofile\AppData\Local) without any problem.
